src/workerd/api/http.c++

```diff
@@ -1195,6 +1196,22 @@ void Request::shallowCopyHeadersTo(kj::HttpHeaders& out) {
 }
 
 kj::Maybe<kj::String> Request::serializeCfBlobJson(jsg::Lock& js) {
+  if(cacheMode != CacheMode::NONE &&
+    util::Autogate::isEnabled(util::AutogateKey::HTTP_REQUEST_CACHE)) {
+    // Works because of the fact there are only two non-none cache modes
+    auto ttl = (CacheMode::NOSTORE == cacheMode) ? -1 : 0;
```

Comment:
I think it would be good to have a switch here so that if we add a new `CacheMode` it will become a compiler error? Or are we not expecting to add any `CacheMode`s ever?